src/utils/utils.ts

```diff
@@ -14,4 +15,26 @@ export async function exists(path: string): Promise<boolean> {
 	}
 }
 
+export async function fileExists(uri: vscode.Uri, filename: string): Promise<boolean> {
+    const path = vscode.Uri.joinPath(uri, filename);
```

Comment:
nit: it seems odd to have two methods in this file named `exists` and `fileExists` doing basically the same thing but with different arguments.  Maybe they should be overloads using the different `fs` implementation based on the parameters provided?